apps/search/style/search.css

```diff
@@ -161,16 +161,16 @@ h2 span {
 
 /* web */
 section.apps {
-  padding-top: 0.5rem;
-  padding-bottom: 1.5rem;
   /* Hide the whitespace between inline elements */
   font-size: 0;
 }
 
 gaia-grid {
   display: block;
-  height: 100%;
+  height: 10rem !important;
```

Comment:
Can we put the height on the containing section and overflow hidden it to avoid the `!important`?